suite-native/module-accounts-import/src/components/XpubHintModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactNode, useState } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { BottomSheet, Box, Button, Text, VStack } from '@suite-native/atoms';
+import { Icon } from '@trezor/icons';
+import { NetworkType } from '@trezor/connect/lib/constants/network';
+import { Video } from '@suite-native/video-assets';
+
+type XpubScanHintSheet = {
+    networkType: NetworkType;
+};
+
+const sheetTriggerStyle = prepareNativeStyle(_ => ({
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+}));
+
+const networkToTitleMap = {
+    bitcoin: 'Where is my public key (XPUB)?',
+    ethereum: 'Where is my receive address?',
+    ripple: 'Where is my receive address?',
+} as const as Record<NetworkType, string>;
```

Comment:
Why do you use type assertion to the use type casting as well? Wouldn't it make sense to just type the object without assertion, if you're restricting type with casting anyway?

Reply:
Should have been `as satisfies ...`. Fixed in 95199a91a.